src/app/frontend/replicasetlist/logsmenu.html

```diff
@@ -0,0 +1,46 @@
+<!--
+Copyright 2015 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<md-menu md-offset="-200 35">
```

Comment:
The numbers -200 35 seem magical. I tried removing them and it looked right. What do you think about removing them for now and later thinking about pixel perfectionism?